app/models/manageiq/providers/azure/cloud_manager/event_catcher/runner.rb

```diff
@@ -49,6 +45,8 @@ def reset_event_monitor_handle
 
   def filtered?(event)
     event_type = parse_event_type(event)
-    filtered_events.include?(event_type)
+    filtered_events.include?(event_type).tap do |result|
+      _log.info "#{log_prefix} Skipping filtered Azure event #{event_type} for #{event["resourceId"]}" if result
+    end
```

Comment:
Just you 😜   `if <cond> then true else false` 👎  

I like this but if you hate tap you could just assign to a variable returned at the end, too.